src/env.mjs

```diff
@@ -15,10 +15,18 @@ export const env = createEnv({
     client: {
         NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY: z.string().min(1),
         NEXT_PUBLIC_DRIVE_LINK: z.string().url().min(1),
+        NEXT_PUBLIC_CLERK_SIGN_IN_URL: z.literal('/signin'),
+        NEXT_PUBLIC_CLERK_SIGN_UP_URL: z.literal('/join'),
+        NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL: z.literal('/'),
+        NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL: z.literal('/'),
```

Comment:
Why do we have to set it in the environment variables if we want it to match anyway? Isn't it a bit redundant?

Reply:
I didn't find any other solution from clerk. I think it's redundant as well...

Reply:
Maybe add a clarifying note that there is no other way from Clerk, as that clears up any confusion surrounding its seemingly redundant nature